ReactWindows/ReactNative.Shared/Views/Text/ReactSpanViewManager.cs

```diff
@@ -110,6 +112,50 @@ public void AddView(DependencyObject parent, DependencyObject child, int index)
 
 #if WINDOWS_UWP
             span.Inlines.Insert(index, inlineChild);
+
+            // Initialize ImportantForAccessibility for the child.
+            // Post on UI thread because the method requires that parent/child
+            // relationship is established but XAML does not set the relationship
+            // until the next loop after the child is added.
+            DispatcherHelpers.RunOnDispatcher(() =>
```

Comment:
>DispatcherHelpers.RunOnDispatcher(() => [](start = 12, length = 39)

Is this deterministic? There's no chance the dispatched action could be called before the view hierarchy is resolved?

Reply:
@rozele Yes, this is guaranteed to be called after. The reason is that the call to span.Inlines.Insert must come on UI thread (since any XAML-related call must be on UI thread) and RunOnDispatcher will post the action on UI thread, it is guaranteed to be executed in the next loop (when the hierarchy will be established).

---
In reply to: [165976725](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r165976725) [](ancestors = 165976725)